source/virtualBuffers/__init__.py

```diff
@@ -257,6 +257,12 @@ def _getParagraphOffsets(self,offset):
 		return lineStart.value,lineEnd.value
 
 	def _normalizeControlField(self,attrs):
+
+		ariaCurrent = attrs.get("IAccessible2::attribute_current")
+		if ariaCurrent != None:
+			attrs['current']= ariaCurrent
+			del attrs["IAccessible2::attribute_current"]

```

Comment:
Probably no need to delete this; we don't do it with other attributes and leaving it there might be useful for debugging.